platform/security/platform-security-core-api/src/main/java/ddf/security/PropertiesLoader.java

```diff
@@ -143,8 +143,8 @@ public static Properties loadProperties(String propertiesFile, ClassLoader class
                 }
                 properties = new Properties();
 
-                try (InputStreamReader reader = new InputStreamReader(new FileInputStream(
```

Comment:
this was accidentally changed during a large find bugs pr, should actually be using propFile